war/src/main/webapp/help/system-config/master-slave/description.html

```diff
@@ -1,11 +1,6 @@
 <div>
-  Optional human-readable description of this slave. This information
-  is displayed on the project configuration screen.
-
+  Optional human-readable description for this agent.
```

Comment:
Unlike any other descriptions in Jenkins, it's not markup-formatted and not shown separately -- instead, it's displayed after the agent name, and should therefore be pretty short. Not sure whether relevant.